src/extension.ts

```diff
@@ -83,102 +86,132 @@ export async function activate(context: vscode.ExtensionContext) {
   function getPRsPerSHAs() {
     // takes the first 22 shas and creates a list to send to the gh api
     let joinedArrayOfSHAs = arrayOfSHAs.slice(0,22).join();
-  
-    octokit
-      .request(`GET /search/issues?type=Commits`, {
-        org: owner,
-        q: joinedArrayOfSHAs,
-      })    
-      .then((octorespSearch: any) => {
-        const issuesBySHAs = octorespSearch.data.items;
-        if (issuesBySHAs.length === 0) {
-          vscode.window.showErrorMessage(
-            "No search results. Try selecting a bigger piece of code or another file."
-          );
-        } else {
-          let issuesWithTitlesAndGroupedComments: {
-            user: any;
-            title: string;
-            comments: any[];
-            created_at: any;
-          }[] = [];
 
-          issuesBySHAs.forEach(async (issue: { url: any }) => {
-            const issueUrl = issue.url;
-            let prTitlesPushed: string[] = [];
+    axios.get('https://app.watermelon.tools/api/github/getIsWithinPlan', {
+      data: {
+        "organizationName": owner
+      }
+    })
+      .then(function (response: any) {
```

Comment:
Any specific reason to wrap it all?
Why not a boolean that awaits?

Reply:
Is it important to change it?

Reply:
No, no prob, my current work will fix it